pkgs/top-level/python-packages.nix

```diff
@@ -32086,6 +32086,26 @@ EOF
 
   treq = callPackage ../development/python-modules/treq { };
 
+  snakeviz = buildPythonPackage rec {
```

Comment:
We have a new policy to put packages into a separate file in `../development/python-modules/` like you see in the example above because python-packages went really big. Could move your module to a separate file as well?